torchdynamo/utils.py

```diff
@@ -292,12 +340,12 @@ def torchscript(model, example_inputs, verbose=True):
         return torch.jit.trace(model, example_inputs)
     except Exception:
         if verbose:
-            log.exception("jit error")
+            log_error("jit error")
```

Comment:
I feel like an exception is a more appropriate thing to log along with its stack trace